ale_linters/markdown/textlint.vim

```diff
@@ -0,0 +1,23 @@
+" Author: tokida https://rouger.info
+" Description: textlint, a proofreading tool (https://textlint.github.io/)
+
+function! ale_linters#markdown#textlint#GetCommand(buffer) abort
+    let l:cmd_path = ale#path#FindNearestFile(a:buffer, '.textlintrc')
+    if !empty(l:cmd_path)
```

Comment:
Write a blank line before `if`. Can `textlint` be run without a configuration file? If it can, maybe you can run it without passing the `-c` option, if no configuration file is found.

Reply:
I tried run with out a configuration file . result, textlint was necessary -c option.